compiler-rt/lib/radsan/tests/radsan_test_context.cpp

```diff
@@ -8,31 +8,42 @@
 
 #include "radsan_test_utilities.h"
 
+#include "radsan.h"
 #include "radsan_context.h"
 #include "radsan_user_interface.h"
 
-TEST(TestRadsanContext, canCreateContext) { auto context = radsan::Context{}; }
+#include <gtest/gtest.h>
 
-TEST(TestRadsanContext, expectNotRealtimeDoesNotDieBeforeRealtimePush) {
+// Test fixture that calls ENSURE_RADSAN_INITED before each test
+class TestRadsanContext : public ::testing::Test {
+protected:
+  void SetUp() override { radsan::EnsureInitialized(); }
```

Comment:
So what happens if this `SetUp` routine is removed?

Reply:
If this is removed, the `flags()` returns a nullptr and we die on line 36 of radsan_user_interface.cpp